src/cloudai/util/docker_image_cache_manager.py

```diff
@@ -251,9 +251,22 @@ def cache_docker_image(
 
         try:
             p = subprocess.run(enroot_import_cmd, shell=True, check=True, capture_output=True)
+            stdout = p.stdout.decode() if p.stdout else ""
+            stderr = p.stderr.decode() if p.stderr else ""
+
+            if "Disk quota exceeded" in stderr or "Write error" in stderr:
+                error_message = (
+                    f"Failed to cache Docker image {docker_image_url}. Command: {enroot_import_cmd}. "
+                    f"Error: '{stderr}'\n\n"
+                    "This error indicates a disk-related issue. Please check if the disk is full or not usable. "
+                    "If the disk is full, consider using a different disk or removing unnecessary files."
```

Comment:
[thought] Do we want to have suggestions right inside error messages? While it is useful, maybe we want to have a single place to map errors to suggestions, like a README file.

Reply:
Let's discuss this with @srinivas212. While it is a good idea, I would like to keep it in the error messages because I don't think users will read README.md. Users tend to report issues or ping us directly.